src/components/Skills/Skills.js

```diff
@@ -46,15 +46,15 @@ const WordClouds = () => {
       aria-hidden="true"
       className="skills__word-clouds"
     >
-      <p style={{ top: 50, left: '100%', fontSize: 23 }}>HTML5</p>
-      <p style={{ top: 0, left: 0, fontSize: 25 }}>WebGL</p>
-      <p style={{ top: 200, left: -60, fontSize: 14 }}>CSS3</p>
-      <p style={{ top: '30%', left: '35%', fontSize: 18 }}>figma</p>
-      <p style={{ top: 40, left: '75%', fontSize: 12 }}>antd</p>
-      <p style={{ top: 380, left: '85%', fontSize: 18 }}>MongoDB</p>
-      <p style={{ top: 320, left: '65%', fontSize: 18 }}>Gatsby</p>
-      <p style={{ top: 350, left: 150, fontSize: 20 }}>ES6</p>
-      <p style={{ top: 120, left: '22%', fontSize: 12 }}>blender</p>
+      <p style={{ top: 50, left: '100%', fontSize: 23, WebkitTouchCallout: "none", WebkitUserSelect: "none", userSelect: "none", msUserSelect: "none" }}>HTML5</p>
```

Comment:
You can just put `user-select: none;` in here

https://github.com/anuraghazra/anuraghazra.github.io/blob/c56c686e772a3326627e68f1864b68f3156b5e3d/src/components/Skills/Skills.js#L29-L36

Reply:
Also no need to put vendor specific CSS.